### PR TITLE
Add santali language

### PR DESCRIPTION
**Pull request by mikehardy**
New translation to Santali language, all credit to Prasanta-Hembram (Rocky_734) on crowdin!

### CODE REVIEW

**Comment by mikehardy** [comment]
Currently no patch release planned for 2.11.2 but I'll marking it for a pick there if we do one, and will of course be in the next 2.12 alpha

File: AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsTest.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
             "de", "el", "en", "eo", "es-AR", "es-ES", "et", "eu", "fa", "fi", "fil", "fr", "fy-NL", "ga-IE", "gl", "got",
             "gu-IN", "he", "hi", "hr", "hu", "hy-AM", "id", "is", "it", "ja", "jv", "ka", "kk", "km", "ko", "ku",
             "ky", "lt", "lv", "mk", "mn", "mr", "ms", "my", "nl", "nn-NO", "no", "pa-IN", "pl", "pt-BR", "pt-PT",
-            "ro", "ru", "sk", "sl", "sq", "sr", "ss", "sv-SE", "sw", "ta", "te", "tg", "th", "ti", "tl", "tn", "tr",
+            "ro", "ru", "sat", "sk", "sl", "sq", "sr", "ss", "sv-SE", "sw", "ta", "te", "tg", "th", "ti", "tl", "tn", "tr",
```
Comment (david-allison-1) [inline AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsTest.java:55]: Note to self: This should have required PREVIOUS_LANGUAGES to change as well.

**Comment by mikehardy** [comment]
@prasanta-hembram we needed a 2.11.2 for other reasons (it's always something! :-) ) so this is headed out to the beta channel as 2.11.2 now and if everything is fine it'll be live in production in ~36 hours. An ongoing basis I don't put out any notice when I resync fresh translation progress from crowdin.com but I definitely do it before each new minor version (2.12.0, 2.13.0 etc) and if at all possible I do it before each patch version as well (2.11.3, 2.11.4 etc) so you may not hear from me again but your work will make it out no problem. Cheers

**Comment by Prasanta-Hembram** [comment]
Thanks you again @mikehardy. I have almost completed half of it, i will complete it soon. I am very thankful to you people for so much hard work for improving this app. Again i thank you for your help, this translation may help Santali tribal Community to engage more in this app. :-)

**Pull request merged**